start_nf.sh

```diff
@@ -18,7 +18,7 @@ cp test/data/*.fastq . && bwa index test/data/ref.fa
 #nextflow run nf_wochenende.nf  -with-timeline -with-report --metagenome testdb --aligner bwamem --remove_mismatching 2 --mq30 --readType PE  --no_dup_removal --no_abra --fastq *R1.fastq
 
 # lisa version with corrected args
-nextflow run nf_wochenende.nf  -with-timeline -with-report --metagenome testdb --aligner bwamem --mismatches 2 --mapping_quality mq30 --readType PE  --no_dup_removal --no_abra --fastq *R1.fastq
+nextflow run nf_wochenende.nf  -with-timeline -with-report --metagenome testdb --aligner bwamem --mismatches 2 --mapping_quality mq30 --readType PE  --no_prinseq --no_abra --fastq *R1.fastq
```

Comment:
of course it can also be run without no_prinseq. no_dup_removal at the moment causes an error, since the `*dup.bam` files aren't created, but they are an expected output by the wochenende process